Update Ember release instructions to cover tool-new-release (#50)
ember-learn/handbook: ember-releases/README.md

# Ember Releases

When a new version of Ember is released,
the learning team has to make sure some parts of our infrastructure are adequately updated to account for it.
**Please update the following in the listed order**:

1. Guides
2. API documentation
3. Release blog post
4. Release pages
5. Glitch Ember starter
6. Ember Wikipedia

The Guides and API docs should be published first, followed by the Release blog post. This way, links to the Guides and API docs can be included in the Release blog post. All other steps come after the blog post.

There are two ways to go about it.

> Consult the relevant document for more detailed instructions:
> 
> 1. [Use tool-new-release](./tool-new-release.md)
> 
> This tool tries to automate the process as much as possible as should be the preferred method.
> 
> 2. [Follow steps manually](./manual.md)
> 
> If you wish to do the process manually, you will find all the steps described here.

Reviewer: **suggestion**:

From lines 17-25, I didn't think it was immediately clear which sentence goes with which of the two options. How about:

```suggestion
Consult the relevant document for more detailed instructions:

1. [Use tool-new-release](./tool-new-release.md). The tool tries to automate the process as much as possible. This option is the preferred method.
2. [Make a release manually](./manual.md).
```
